nodes/households/households_flexibility_p2p_electricity.converter.ad

```diff
@@ -3,7 +3,7 @@
 - output.electricity = 0.85
 - output.loss = elastic
 - groups = [cost_flexibility, lv_net_flexibility, p2p]
-- merit_order.group = power_to_power
+- merit_order.group = household_batteries
```

Comment:
I'll need to write a migration in ETEngine to update existing scenarios to use this new name.

Reply:
Okay, good point! Forgot about that.